### DISCUSSION TITLE
5.0.2 issue with a theme - Popper.js

### DISCUSSION THREAD

**Discussion by eurobank** [mindstellar/shopclass#377] [Q&A] [answered]
After 5.0.2. upgrade i see an error in my console (f12) when in the ad post page. 

```
Uncaught TypeError: Bootstrap tooltips require Popper.js (https://popper.js.org)
    a https://www.website.eu/oc-content/themes/osclasswizards/js/bootstrap.min.js:6
```

The theme is Osclasswizards 2.0.6

Any ideas what this could be and how to solve it?

**Comment by eurobank** [comment, marked as answer]
closed.

**Comment by navjottomer** [comment]
It is saying that you have to include popper.js on your page. Sorry, will not be available for any help for 1-2 days.

**Reply by eurobank** [reply]
Yes, we did find that, it was a mistake trying to make a theme to work right. We commented out a wrong line of code.

thank you !
